src/xrpld/rpc/detail/TransactionSign.cpp

```diff
@@ -710,6 +709,104 @@ transactionFormatResultImpl(Transaction::pointer tpTrans, unsigned apiVersion)
 
 //------------------------------------------------------------------------------
 
+XRPAmount
+getBaseFee(Application const& app, Config const& config, Json::Value tx)
```

Comment:
`tx` value modified here, shouldn't it be passed as ref here and through `getCurrentFee`